api/firmware/bitboxbase_test.go

```diff
@@ -25,7 +25,7 @@ import (
 func TestBitBoxBaseConfirmPairing(t *testing.T) {
 	testConfigurations(t, func(env *testEnv, t *testing.T) {
 		env.onRequest = func(*messages.Request) *messages.Response {
-			return responseSuccess
+			return responseSuccessMessage
```

Comment:
Didn't see where this was defined and looked like a non-test, regular global var... Consider renaming to something like `testEnvDeviceResponseSuccessMessage` or `testDeviceResponseOK`.